Flag trigger + audit logs (#39)
bucketeer-io/bucketeer-docs: docs/feature-flags/audit-logs.mdx

---
title: Audit Logs
# sidebar_position:
slug: /feature-flags/audit-logs
description: Use the flag triggers to automatically turn feature flags on or off by calling a URL using the POST method.
tags: ['log', 'audit-logs']
---

The Audit Logs feature stores all log history from the current Environment. You can use the logs to identify problems, or who performed modification in your project flags. 

> To access the logs associated with the current environment, select **Audit Logs** on the dashboard sidebar. To check logs from other environments, you first need to change to the desired environment and then access the Audit Logs page.

Reviewer: We could simplify this.

`To access the logs associated with the current environment, select Audit Logs on the dashboard sidebar.` -> `To check the logs, access the Audit Logs page from the sidebar menu.`

---
title: Audit Logs
# sidebar_position:
slug: /feature-flags/audit-logs
description: Use the flag triggers to automatically turn feature flags on or off by calling a URL using the POST method.
tags: ['log', 'audit-logs']
---

import CenteredImg from '@site/src/components/centered-img/CenteredImg';

> The Audit Logs feature stores all log history from the current Environment. You can use the logs to identify problems, or who performed modification in your project flags. 

Reviewer: ```suggestion
The Audit Logs feature stores all log history from the current Environment. You can use the logs to identify problems or who performed modifications in your project flags.
```

To check the logs, access the **Audit Logs** page from the sidebar menu. To check logs from other environments, you first need to change to the desired environment and then access the Audit Logs page.

<CenteredImg
  imgURL="img/feature-flags/audit-logs/overview.png"
/>

## Filtering logs

By default, the Audit Logs displays the most recent entries. However, you can use the **Sort** button at the top right corner to switch to the **Oldest** first if you need.

> To make it easier to inspect and find problems source, the Bucketeer system provides several filters you can use to find desired logs. The complete filter options is listed below:

Reviewer: ```suggestion
To make it easier to inspect and find problem sources, the Bucketeer system provides several filters you can use to find desired logs. The complete filter options are listed below:
```

> - **Email**: User email who perform the changed that originated the log, such as changing a flag variation, for example.

Reviewer: ```suggestion
- **Email**: User email who performed the change, such as changing a flag variation.
```

- **Date**: Specify a date range when the logs were created.

<CenteredImg
  imgURL="img/feature-flags/audit-logs/date.png"
/>

- **Entity**: Select the type of entity related to the log source. You can use this option to filter logs related only to changes in Goals, for example. The complete list is presented next:
    - Feature Flag
    - Goal
    - Experiment
    - User Segment
    - Account
    - API Key
    - Auto Operation
    - Progressive Rollout
    - Push
    - Subscription

<CenteredImg
  imgURL="img/feature-flags/audit-logs/action.png"
/>

### Actions

> Several actions can create a log in Bucketeer. The toggle below contain a table listing all possible action that can generate new logs. 

Reviewer: ```suggestion
Several actions can create a log in Bucketeer. The toggle below contains a table listing all possible actions to generate new logs.
```
